src/app/admin/requests/request-details/request-address-field/request-address-field.component.ts

```diff
@@ -33,37 +35,35 @@ export class RequestAddressFieldComponent implements OnInit {
           coors: [47.02486150651041, 28.832740004203416],
           address: 'Arcul de Triumf',
         },
-        panelClass: 'cdk-overlay-pane-no-padding',
+        panelClass: 'esri-map',
```

Comment:
What is the reason of changing this class?
This class removes the padding of the dialog. If the invention was to have dialog padding, that this `panelClass` can be removed at all

Reply:
Reverted back to cdk-overlay-pane-no-padding.